### PR TITLE
[15.0][MIG] purchase_order_uninvoiced_amount: Migration to version 15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT36510

ping @carlosdauden @victoralmau

### CODE REVIEW

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields
+from odoo.tests.common import Form, SavepointCase
+
+
+class TestPurchaseOrderUninvoiceAmount(SavepointCase):
+    def setUp(self):
+        super(TestPurchaseOrderUninvoiceAmount, self).setUp()
```
Comment (victoralmau) [inline purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py]:
```suggestion
from odoo.tests.common import Form, TransactionCase


class TestPurchaseOrderUninvoiceAmount(TransactionCase):
    def setUp(self):
        super().setUp()
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Please rebase to solve some test warnings.

**Comment by sergio-teruel** [comment]
@victoralmau Changes done!!

**Comment by victoralmau** [comment]
Tests fail, could you check it?

**Comment by sergio-teruel** [comment]
Yes.. I know...

**Comment by sergio-teruel** [comment]
Tests fixed!!

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        )
+        # Products
+        self.product_category = self.env["product.category"].create(
+            {"name": "Test Product category"}
+        )
+        self.product_1 = self.env["product.product"].create(
+            {
+                "name": "Test Sale Product",
+                "sale_ok": True,
+                "type": "consu",
```
Comment (victoralmau) [inline purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py:48]:
```suggestion
                "detailed_type": "consu",
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

File: purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py (hunk 2 above)
Comment (sergio-teruel) [reply purchase_order_uninvoiced_amount/tests/test_purchase_order_uninvoiced_amount.py:48]: @victoralmau  at this time is not needed this change..

**Review by victoralmau — APPROVED** [review]
Code review OK

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**